frontend/src/components/pages/VolunteerScheduling/index.tsx

```diff
@@ -21,27 +34,74 @@ interface UseStepType {
   navigation: NavigationProps | any;
 }
 
-const VolunteerScheduling = () => {
-  const { step }: UseStepType = useStep({
+const shiftDefaultData = ({
+  id: "",
+  donorId: "",
+  isPickup: "",
+  pickupLocation: "",
+  volunteerTime: "",
+  categories: [],
+  size: "",
+  dayPart: "",
+  startTime: "",
+  endTime: "",
+  notes: "",
+  type: "",
+} as unknown) as CheckInWithShiftType | ScheduleWithShiftType;
+
+const VolunteerScheduling = (shiftData = shiftDefaultData) => {
+  const [volunteerStatus, setVolunteerStatus] = useState<Status>();
+  const { authenticatedUser } = useContext(AuthContext);
+  const [shiftId, setShiftId] = useState<string>("1");
```

Comment:
any idea what we want this to be the default?

Reply:
why is this `setShiftId` being passed into different components here?
what is this value?

Reply:
i think jessie mentioned that it was to set either the checkIn id or the scheduling id but im not sure if it needs to be done like this let me see if i can refactor it